Actions/settings.schema.json

```diff
@@ -0,0 +1,592 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "description": "Schema for AL-Go settings",
+    "type": "object",
```

Comment:
Should we consider having a repository schema file, a project schema and a workflow schema?
I mean a setting like "workflowSchedule" is only allowed in workflow specific settings files f.ex.
and other settings are supposed to be in repo settings only.

Reply:
Thanks for bringing it up. I did think of that, but decided to keep it loose and don't check specific settings files and settings layers, but the settings object after all _layers_ are applied (org settings, repo settings, project settings etc.).

If we don't guard the specific settings level, there's much more flexibility on how to the settings can be organized. In a single-project setup, for example. all settings can be in AL-Go-settings.json (I think empty .AL-Go/settings.json might still be needed though).

About `workflowSchedule`, why not allowing the _default_ value to be set in a single place (e.g. the org settings) and only changed/removed on specific cases.

There are a lot of different setups, and my thinking is that allowing flexibility with AL-Go settings will pay off.